docs/build/html/_sources/quickstart.rst.txt

```diff
@@ -84,23 +84,26 @@ GoogleDrive treats everything as a file and assigns different mimetypes for diff
       return newFolder
 
 
-Get Id of the title
--------------------
+Return File ID via File Title
+-----------------------------
 
-``get_id_of_title`` is a function that return id of the given title's file in the directory.
+A common task is providing the Google Drive API with a file id.
+``get_id_of_title`` is a function that returns the id of a file handle by searching the file titles in a
+given directory. The function takes two arguments, title and parent_directory_id. Title is a string that
+will be compared against file titles included in a directory identified by the parent_directory_id.
 
 .. code-block:: python
 
     def get_id_of_title(title,parent_directory_id):
       foldered_list=drive.ListFile({'q':  "'"+parent_directory_id+"' in parents and trashed=false"}).GetList()
```

Comment:
```
    def _get_remote_item_ids(self, parent_ids, title):
        if not parent_ids:
            return None
        query = "trashed=false and ({})".format(
            " or ".join(
                f"'{parent_id}' in parents" for parent_id in parent_ids
            )
        )
        query += " and title='{}'".format(title.replace("'", "\\'"))

        # GDrive list API is case insensitive, we need to compare
        # all results and pick the ones with the right title
        return [
            item["id"]
            for item in self._gdrive_list(query)
            if item["title"] == title
        ]
```

that's what we have in the DVC (dvc.org) code that is main user of the PyDrive2 now. It looks better a me.

Reply:
okay, probably too complicated for this doc :( disregard please, may be a link to this code can be useful though.